webservices/docs.py

```diff
@@ -21,7 +21,7 @@
 If you would like to use the FEC's API programmatically, you can sign up for your own API key using our form. Alternatively, you can still try out our API without an API key by using the web interface and using DEMO_KEY. Note that when you use the openFEC API you are subject to the [Terms of Service](https://github.com/fecgov/FEC/blob/master/TERMS-OF-SERVICE.md) and [Acceptable Use policy](https://github.com/fecgov/FEC/blob/master/ACCEPTABLE-USE-POLICY.md).
 
 
-Signing up for an API key will enable you to place up to 1,000 calls an hour. Each call is limited to 100 results per page. You can email questions, comments or a request to get a key for 7,600 calls per hour (120 calls per minute) to <a href="mailto:APIinfo@fec.gov">APIinfo@fec.gov</a>. You can also ask questions and discuss the data in a community led [group](https://groups.google.com/forum/#!forum/fec-data).
+Signing up for an API key will enable you to place up to 1,000 calls an hour. Each call is limited to 100 results per page. You can email questions, comments or a request to get a key for 7,600 calls per hour (120 calls per minute) to <a href="mailto:APIinfo@fec.gov">APIinfo@fec.gov</a>. You can also ask questions and discuss the data in a community led [group](https://groups.google.com/forum/#!forum/fec-data). Occasionally the Federal Election Commission (FEC) will decide to deprecate, including making breaking changes to an API endpoint, feature, or service. Please see our [deprecation policy](https://github.com/fecgov/openFEC/wiki/Deprecation-policy) for more information.
```

Comment:
I don't think we need to spell out the FEC in the new paragraph when we've referred to FEC several times before this point. 

We also try not to use the words see or view whenever possible because it is not inclusive to individuals with visual impairments.

Reply:
@dorothyyeager Please see my above comment. Would appreciate your thoughts on the text changes here.

Reply:
Normal custom is to spell out the name the first time it's used, followed by the acronym in parentheses. So if Federal Election Commission is spelled out above in the text, we can use just "FEC" here.

And yes, words we avoid include "see," "view," "above," and "below."